lib/web/apiserver_test.go

```diff
@@ -4700,14 +4707,20 @@ func TestGetWebConfig_WithEntitlements(t *testing.T) {
 	expectedCfg.Entitlements[string(entitlements.JoinActiveSessions)] = webclient.EntitlementInfo{Enabled: false}
 	expectedCfg.Entitlements[string(entitlements.K8s)] = webclient.EntitlementInfo{Enabled: false}
 
-	// request and verify enabled features are enabled.
-	re, err = clt.Get(ctx, endpoint, nil)
-	require.NoError(t, err)
-	require.True(t, strings.HasPrefix(string(re.Bytes()), "var GRV_CONFIG"))
-	str = strings.ReplaceAll(string(re.Bytes()), "var GRV_CONFIG = ", "")
-	err = json.Unmarshal([]byte(str[:len(str)-1]), &cfg)
-	require.NoError(t, err)
-	require.Equal(t, expectedCfg, cfg)
+	// request and verify enabled features are eventually enabled.
+	require.EventuallyWithT(t, func(c *assert.CollectT) {
```

Comment:
```suggestion
	require.EventuallyWithT(t, func(t *assert.CollectT) {
```

This makes it impossible for you to accidentally use `t` instead of `c` (which is what you're doing incorrectly below).

Reply:
I would still rename `c` to `t` so that you shadow the original value of `t`.

Reply:
Thanks, I misread it the first time. Done in https://github.com/gravitational/teleport/commit/690dd2771eea39cdd4c96309743b1ed767023849.